example.d

```diff
@@ -30,8 +31,14 @@ void main()
     log.warn("mostly harmless"d);
     log.info("the answer is %s", 42);
     log.info!"the answer is %s"(42);
+    static if (version_minor >= 108)
+    {
+        // mixin so that it passes the lexer
+        mixin(`log.info(i"the answer is $(42)");`);
+    }
     log.trace(details);
 
+
```

Comment:
The second blank line is already here!
Move the complicated `static if` down here.

Reply:
done